<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>publish_article_interface</class>
 <widget class="QWidget" name="publish_article_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>651</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_publish">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发布</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>发布版块:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_article_type">
       <item>
        <property name="text">
         <string>日常分享</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>悬赏接单</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>失物招领</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>经验分享</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>二手市场</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
